src/components/SingleSelect/SingleSelect.jsx

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import _ from 'lodash';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition } from '../../util/component-definition';
+import * as reducers from './SingleSelect.reducers';
+import DropMenu from '../DropMenu/DropMenu';
+import CaretIcon from '../Icon/CaretIcon/CaretIcon';
+
+const boundClassNames = lucidClassNames.bind('&-SingleSelect');
+
+const {
+	any,
+	bool,
+	func,
+	node,
+	number,
+	object,
+	shape,
+	string
+} = React.PropTypes;
+
+/**
+ *
+ * {"categories": ["controls", "selectors"], "madeFrom": ["DropMenu"]}
+ *
+ * A selector control (like native `<select>`) which is used to select a single option from a dropdown list.
+ * Supports option groups with and without labels.
+ */
+
+const SingleSelect = React.createClass(createLucidComponentDefinition({
+	displayName: 'SingleSelect',
+
+	reducers,
+
+	childProps: {
+		Placeholder: null,
+		Option: DropMenu.Option.propTypes,
+		OptionGroup: DropMenu.OptionGroup.propTypes
+	},
+
+	propTypes: {
+		/**
+		 * Should be instances of {`SingleSelect.Placeholder`, `SingleSelect.Option`, `SingleSelect.OptionGroup`}. Other direct child elements will not render.
+		 */
+		children: node,
+		/**
+		 * Appended to the component-specific class names set on the root elements. Applies to *both* the control and the flyout menu.
+		 */
+		className: string,
+		/**
+		 * Styles that are passed through to root element.
+		 */
+		style: object,
+		/**
+		 * Allows user to reset the `optionIndex` to `null` if they select the placeholder at the top of the options list.
+		 * If `false`, it will not render the placeholder in the menu.
+		 */
+		allowReset: bool,
```

Comment:
Shouldn't this be `isAllowReset` as per our rules around naming boolean props? My desire for props that read like decent English doesn't sound so silly anymore, does it? ;)